internal/handlers/common/filter.go

```diff
@@ -495,6 +495,13 @@ func filterFieldExpr(doc *types.Document, filterKey string, expr *types.Document
 				return false, err
 			}
 
+		case "$elemMatch":
```

Comment:
Should `$elemMatch` be before `$size` in this list?

Reply:
Moved it.